.vscode/settings.json

```diff
@@ -17,6 +17,7 @@
   "eslint.enable": false,
   "cSpell.allowCompoundWords": true,
   "cSpell.caseSensitive": true,
+  "rust-analyzer.checkOnSave": false,
```

Comment:
should remove this

Reply:
yeah, I mark this as a draft, I will recover it when it is ready to merge, I found that vscode becomes pretty lag when enabling this.